spring-kafka-test/src/main/java/org/springframework/kafka/test/EmbeddedKafkaZKBroker.java

```diff
@@ -104,6 +104,8 @@ public class EmbeddedKafkaZKBroker implements EmbeddedKafkaBroker {
 
 	public static final int DEFAULT_ZK_CONNECTION_TIMEOUT = DEFAULT_ZK_SESSION_TIMEOUT;
 
+	private static final String NUM_PARTITIONS = "num.partitions";
```

Comment:
This is broker property.
It has nothing to do with the client application.
Why would one need to have it as a top-level API, since this is only a part if this class?
There is a dedicate constructor argument for this config.

On the other hand, with Apache Kafka `4.0` we are aim to remove Zookeeper support from the project.
So, your effort for this property will be eliminated.

Please, elaborate the reasoning for such a change.